### DISCUSSION TITLE
2FA

### DISCUSSION THREAD

**Discussion by WillMontgomery** [diced/zipline#229] [Ideas]
Authenticator-style 2FA would be a welcomed feature :) Just putting this in writing so it doesn't get forgotten. Thanks for all you do!

**Comment by diced** [comment]
Thanks! Already started working on the backbones yesterday, should be out to upstream maybe by the next week.

**Comment by diced** [comment]
Those who want to give it a try, the feature is now available in this commit: https://github.com/diced/zipline/commit/31759111056d4ea97cf8f7844bf210a86a25e47a. Documentation available here: https://zipline-trunk.diced.tech/docs/guides/totp https://zipline-trunk.diced.tech/docs/config/mfa
